sceptre/scipool/templates/sc-enduser-iam.yaml

```diff
@@ -32,6 +33,7 @@ Resources:
       ManagedPolicyArns:
         - arn:aws:iam::aws:policy/AWSServiceCatalogEndUserFullAccess
         - !Ref SCEnduserExpandedPolicy
+        - !Ref SCCostExplorerPolicy
```

Comment:
This role is for non sage staff users.  I'm not sure if should provide external users access to cost explorer?  Your thoughts @brucehoff ?

Reply:
This is a issue that I haven't thought about:  The goal is to give Sage workers (not all Service Catalog users) the ability to track costs.   As @zaro0508  suggests, we should NOT be giving non-Sage workers access to Cost Explorer and, therefore we should not be sending them the same spend alerts.  Will elaborate in https://sagebionetworks.jira.com/browse/IT-1532